vms/platformvm/service_test.go

```diff
@@ -441,7 +441,7 @@ func TestGetStake(t *testing.T) {
 		}
 		response := GetStakeReply{}
 		require.NoError(service.GetStake(nil, &args, &response))
-		require.EqualValues(uint64(defaultWeight), uint64(response.Staked))
+		require.Equal(uint64(defaultWeight), uint64(response.Staked))
```

Comment:
nit: I'd change defaultWeight type to uint64 (it's int currently) so to avoid a whole bunch of these casts